--- node_modules/gitbook-plugin-mathjax-commonhtml/index.js	2016-12-20 17:11:57.000000000 +0900
+++ node_modules_new/gitbook-plugin-mathjax-commonhtml/index.js	2017-05-21 16:47:24.000000000 +0900
@@ -1,7 +1,7 @@
 'use strict';
 
 const fse = require('fs-extra');
-const mjAPI = require('mathjax-node/lib/mj-single.js');
+const mjAPI = require('mathjax-node/lib/main.js');
 const path = require('path');
 
 const pluginName = 'mathjax-commonhtml';
